src/test/create-user.ts

```diff
@@ -0,0 +1,51 @@
+import axios from 'axios';
+import { UserInput } from '../resolvers';
+import { dataSource } from '../data-source';
+import { User } from '../user';
+
+describe('Testing createUser Mutation', () => {
+  it('should return created user on test db', async () => {
```

Comment:
```suggestion
  it('should return created user', async () => {
```

Acho que não é necessário dizer que é no test db. O intuito do teste é verificar se a mutation está funcionando corretamente. A informação de qual ambiente ou banco isso está acontecendo não é relevante.